db/migrate/20221109173708_create_foods.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFoods < ActiveRecord::Migration[6.1]
+  def change
+    create_table :foods do |t|
+      t.references :footprint, foreign_key: true
+      t.float :min_carbon_footprint, null: false, default: 0
+      t.float :max_carbon_footprint, null: false, default: 0
```

Comment:
We need to add the other fields as well, see the suggestion from the API side, please.